abi/abit/types.go

```diff
@@ -180,6 +182,10 @@ var (
 		Name: "uint64",
 		Kind: S,
 	}
+	Uint120 = Type{
```

Comment:
When is `uint120` used?

Reply:
One of the production contracts I have been testing has a Uint120. I didn't ask any questions. Just added the type.